version_tool.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+
+if len(sys.argv) != 3:
+    print('original version number and new version number are both needed!')
+else:
+    origin_version, new_version = sys.argv[1], sys.argv[2]
+
+    # file_path contains all files'(README, code, documentation source) paths that may contain version number.
+    file_path = ['README.md', 'setup.py']
+
+    # Walk through directories to add all corresponding files' paths into file_path
+    for root, _, files in os.walk('./pydatastructs'):
```

Comment:
May be we can re-use the following function to list all the files,

https://github.com/codezonediitj/pydatastructs/blob/97cb9fdd177f11b763c3f72ead11b4ba525ce4ed/pydatastructs/utils/tests/test_code_quality.py#L4

Reply:
yeah, i see. But the root_path in `_list_files` doesn't make sense since it will go to '../../cur_file_path'.

So I try to modify the `_list_files` to pass one more argument, like `root_path=None`, so we can specify the root_path at will. 

Then one another issue occurred, you can see from the issue: ImportError #479

Reply:
> So I try to modify the `_list_files` to pass one more argument, like `root_path=None`, then we can specify the root_path at will.
> 
> Does adding such a modification in `_list_files` make sense?

Sounds good to me. `_list_files` is anyway a private function, so changing its API is not a problem.